setup.py

```diff
@@ -130,7 +122,7 @@ def cythonize(extensions, **_ignore):
           author_email='t.lavergne@met.no',
```

Comment:
Thomas isn't active anymore, this should be changed

Reply:
Do you want me to switch it to your email?

Reply:
or to the core-devs email ?